plugins/org.eclipse.emfcloud.modelserver.glsp.integration/META-INF/MANIFEST.MF

```diff
@@ -19,7 +19,9 @@ Require-Bundle: org.eclipse.glsp.server,
  org.eclipse.elk.alg.layered;bundle-version="0.7.0",
  org.eclipse.emf.ecore,
  org.eclipse.emfcloud.modelserver.glsp.notation.model,
- org.eclipse.emfcloud.modelserver.glsp.notation.commands
+ org.eclipse.emfcloud.modelserver.glsp.notation.commands,
+ org.apache.logging.log4j;bundle-version="2.17.1",
+ org.apache.log4j;bundle-version="1.2.19"
```

Comment:
do we need both log4j versions?

Reply:
Yes that's unfortunate - GLSP has not yet upgraded to log4j 2 and needs the 1.x version. And we provide both GLSP and ModelServer classes in this package. We can drop it once GLSP has upgraded as well I'd say.